doc/api/http2.md

```diff
@@ -3010,6 +3010,14 @@ properties.
   meaningful if sent by the server. Once the `enableConnectProtocol` setting
   has been enabled for a given `Http2Session`, it cannot be disabled.
   **Default:** `false`.
+* `customSettings` {Object} Specifies additional settings, yet not implemented
+  in node and the underlying libraries. The key of the object defines the
+  numeric value of the settings type (as defined in the "HTTP/2 SETTINGS"
+  registry established by \[RFC 7540]) and the values the actual numeric value
+  of the settings.
+  It is only supported for sending SETTINGS.
+  Custom settings are not supported for the functions retrieving remote and
+  local settings as nghttp2 does not pass unknown HTTP/2 settings to Node.js.
```

Comment:
This should indicate that there are a maximum number of custom settings allowed. It should also indicate the acceptable numeric value range of both the key and the value.

Reply:
I have added something and it is now pushed. (Still no jslinting, I try to recompile the next hours and hopefully than the jslinter works).